CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ For information on changes in released versions of Teku, see the [releases page]
  - Upgraded jdk16 and default docker image to use eclipse-tumerin builds of OpenJDK.
  - jdk14 and jdk15 docker images have been upgraded to use the latest Ubuntu. Note that these images will be removed in future versions.
  - Reduced memory usage and GC pressure created while tracking the latest attestations for each validator.
+ - Logged a message to indicate when the node starts sync.
```

Comment:
nit: I'd probably consolidate the start / stop sync changelog since they're both really the 'same thing'